build.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import fs from "fs";
+import archiver from "archiver";
+
+// Remove the existing build files
+fs.rmSync('./dist', {
+    force: true,
+    recursive: true,
+});
+
+// Create directory to store build artifects
+fs.mkdirSync('./dist');
+fs.copyFileSync("run.sh", "./dist/run.sh");
+
+// Specity the output and initialize the archiver
```

Comment:
Did you mean "Specify" here?
```suggestion
// Specify the output and initialize the archiver
```

Reply:
Fixed the typo.

https://github.com/dhruvdhaduk-simform/img-magnifier-extension/pull/1/commits/6075e51df74a4f4c80774c70a5f73a32beb87727